lib/spack/docs/.gitignore

```diff
@@ -5,3 +5,4 @@ llnl*.rst
 _build
 .spack-env
 spack.lock
+_spack_root
```

Comment:
Building the docs creates this symlink, which then ends up in git's stage directory